### PR TITLE
Getting back to production!

### LINKED ISSUE #190: Remove all MVP related code
Remove all unnecessary code for PC Production

### LINKED ISSUE #199: README Go build
Go build command should be within a code block

### LINKED ISSUE #208: Readme config is outdated
`--dbreigion` is new

### LINKED ISSUE #214: Define a simple message FlatBuffers schema
````
Message {
  type
  payload
  timestamp
}
````

This will be the basic building blocks of the P2P messages

### LINKED ISSUE #215: Define a simple 'Hello' message FBS schema
Goal: Send node info over the wire, doesn't have to be the production message
````
helloMessage{
  gid // this is all for now
}
````

### LINKED ISSUE #216: Develop a SIMPLE, serialization independent, messaging interface to execute P2P messages

### LINKED ISSUE #217: Test serialization/deserialization of message type (flatbuffers)

### PR DESCRIPTION
**Pull request by andrewnguyen22**
If you want to compile your own .fbs files must install compiler ./flatc 

closes #214 
closes #215 
closes #217 
closes #216 
closes #190 
closes #199 
closes #208

### CODE REVIEW

**Comment by andrewnguyen22** [comment]
This PR is getting quite large @iajrz just a reminder to review at your convenience

**Review by iajrz — COMMENTED** [review]
amazing amount of changes; can't wait for CI/CD to be 100% operational to fully test how things go.

**Comment by andrewnguyen22** [comment]
@iajrz indeed, please approve if you are satisfied with the code

**Review by iajrz — APPROVED** [review]

**Pull request merged**
